src/Core/functions.php

```diff
@@ -16,6 +16,10 @@
  */
 namespace Cake\Core;
 
+use Cake\I18n\Date;
```

Comment:
This makes the i18n package a dependency of the core package, don't really like that. Perhaps these `toDate(Time)` functions should go into `I18n/functions.php` instead.

Reply:
I agree that having the core depend on i18n instead of the other way around is a bit concerning, so I took a quick look at `i18n/functions.php`. All of those functions are prefixed by a double-underscore, but I think it would be unusual to repeat that pattern for these two functions (not that you suggested that in particular).

So, would it be appropriate to move these functions to the other `functions.php` but NOT do the double-underscore prefix naming convention? Or is that a hard rule?

Reply:
There's no need for new functions added to the I18n/functions.php file also have underscore prefix unless they are performing a similar task :)

Reply:
The `__` on those functions is a convention inherited from [gettext](https://www.php.net/manual/en/function.-.php) which provides `_`. Our non-extension implementation of po/mo files uses `__` so the other text translation functions also use `__` 🤷 . However, new functions need not bother with that.

:+1: on ADmad's suggestion of putting these functions into `i18n/functions.php`. They'll still be autoloadable but keeps our internal dependencies simpler.